### PR TITLE
Move gitdata to art-common

### PR DESCRIPTION
**Pull request by locriandev**

### CODE REVIEW

File: elliott/elliottlib/gitdata.py [hunk 1]
```diff
@@ -1,262 +0,0 @@
...
-        if not os.path.isdir(self.data_dir):
-            raise GitDataPathException('{} is not a valid sub-directory in the data'.format(self.sub_dir))
-
-    def bz(self):
-        return self.load_data(key='bug')
-
-    def bz_server_url(self):
-        return self.bz().data['server']
-
-    def bz_target_release(self):
```
Comment (thegreyd) [inline elliott/elliottlib/gitdata.py:180]: I double checked and these aren't used anywhere so they are good to be deleted

Comment (locriandev) [reply elliott/elliottlib/gitdata.py:180]: Cool, going to merge this and set up a new PR to remove the obsolete methods

**Pull request merged**